sql/moz-fx-data-shared-prod/firefox_accounts_derived/docker_fxa_customs_sanitized_v1/metadata.yaml

```diff
@@ -8,9 +16,11 @@ labels:
   application: fxa
   incremental: true
   schedule: daily
-scheduling:
-  dag_name: bqetl_fxa_events
-  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']
+# descheduled as this table was populated using data arriving from the AWS deployment,
+# since the migration there is no need data arriving that needs to be copied.
```

Comment:
This wording doesn't quite make sense.  Did you perhaps mean this:
```suggestion
# since the migration there is no new data arriving that needs to be copied.
```